src/services/network-modification-types.ts

```diff
@@ -37,3 +48,307 @@ export interface VscModificationInfo {
     connectionPosition?: string | null;
     terminalConnected?: boolean | null;
 }
+
+type AttributeModification<T> = { value: T; op: string };
```

Comment:
this type is already defined in converter-station-utils.tsx. may be you can remove one of the definitions ?

Reply:
and move it to a more generic place (for cyclic dependencies)

Reply:
> converter-station-utils.tsx

Done.